gax-java/gax/src/main/java/com/google/api/gax/rpc/ClientContext.java

```diff
@@ -325,6 +348,8 @@ public abstract static class Builder {
     @BetaApi("The surface for tracing is not stable yet and may change in the future.")
     public abstract Builder setTracerFactory(ApiTracerFactory tracerFactory);
 
+    public abstract Builder setGdchApiAudience(String credentialsApiAudience);
```

Comment:
It's better to keep the field name and getter name consistent.

Reply:
Done